File: DAL/Entities/EmployeeVacation.cs
using System;

namespace DAL
{
    public enum VacationType
    {
        VacationLeave = 1,
        SickLeave = 2,
        Holiday = 3
    }

    public class EmployeeVacation
    {
        public int IDEmployeeVacation { get; set; }
        public string EmployeeFirstName { get; set; }
        public string EmployeeLastName { get; set; }
        public VacationType Leave { get; set; }
        public DateTime From { get; set; }
        public DateTime To { get; set; }

        public EmployeeVacation(string employeeFirstName, string employeeLastName, VacationType leave, DateTime from, DateTime to)
        {
            EmployeeFirstName = employeeFirstName;
            EmployeeLastName = employeeLastName;
            Leave = leave;
            From = from;
            To = to;
        }

        public EmployeeVacation(int id, string employeeFirstName, string employeeLastName, VacationType leave, DateTime from, DateTime to)
        {
            IDEmployeeVacation = id;
            EmployeeFirstName = employeeFirstName;
            EmployeeLastName = employeeLastName;
            Leave = leave;
            From = from;
            To = to;
        }
    }
}

File: calendarFront/src/app/entities/employeeVacation.ts
export enum VacationType{
    VacationLeave = 1,
    SickLeave = 2,
    Holiday = 3
}

export class EmployeeVacation{
    IDEmployeeVacation: Number;
    EmployeeFirstName: String;
    EmployeeLastName: String;
    vacationType : VacationType;
    From: Date;
    To: Date;

    constructor(id: Number, firstName: String, lastName: String, vacationType:VacationType, from: Date, to:Date){
        this.IDEmployeeVacation = id;
        this.EmployeeFirstName = firstName;
        this.EmployeeLastName = lastName;
        this.vacationType = vacationType;
        this.From = from;
        this.To = to;
    }
}